CRM/Contribute/Form/AdditionalPayment.php

```diff
@@ -392,79 +398,29 @@ public function submit($submittedValues) {
     CRM_Core_Session::setStatus($statusMsg, ts('Saved'), 'success');
   }
 
-  /**
-   * @param $submittedValues
-   */
-  public function processCreditCard($submittedValues) {
+  public function processCreditCard() {
     $config = CRM_Core_Config::singleton();
     $session = CRM_Core_Session::singleton();
 
-    $unsetParams = array(
-      'trxn_id',
-      'payment_instrument_id',
-      'contribution_status_id',
-    );
-    foreach ($unsetParams as $key) {
-      if (isset($submittedValues[$key])) {
-        unset($submittedValues[$key]);
-      }
-    }
-
-    // Get the required fields value only.
-    $params = $submittedValues;
-
```

Comment:
I haven't made sense of this part - if you have reviewed it I'm fine with that since I know Pradeep concluded it was cruft